gate-web/src/test/groovy/com/netflix/spinnaker/gate/controllers/WebhookControllerSpec.groovy

```diff
@@ -91,4 +91,16 @@ class WebhooksControllerSpec extends Specification {
     NestedServletException ex = thrown()
     ex.message.startsWith("Request processing failed; nested exception is retrofit.RetrofitError: Failed to connect to localhost")
   }
+
+  void 'handles CDEvents API with BAD_REQUEST'() {
```

Comment:
Could we get a test for the happy path (something like a CDEvent webhook is received, it calls the `echoService`?

Reply:
yes adding test for happy path..